### PR TITLE
Issue1129 energy plus zone

### LINKED ISSUE #1129: add model to interface with EnergyPlus thermal zone model
This branch is to add the Modelica code to interface with the EnergyPlus thermal zone model.
See https://lbl-srg.github.io/soep/softwareArchitecture.html#coupling-of-energyplus-envelope-and-room-with-modelica-based-hvac-and-control

To connect to HVAC, the model `Buildings.Fluid.FMI.ExportContainers.HVACZone` may be used.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1129.

### CODE REVIEW

**Comment by mwetter** [comment]
Todo:

- [ ] Update release notes.
- [ ] Update overview table of users' guides.

**Comment by mwetter** [comment]
@JayHuLBL : This is ready for review.

File: Buildings/Fluid/FixedResistances/PressureDrop.mo [hunk 1]
```diff
@@ -10,7 +10,10 @@ model PressureDrop
                   Dialog(group = "Transition to laminar",
                          enable = not linearized));
 
-  final parameter Real k = if computeFlowResistance then
+  // Below is a temporary fix for Dymola 2020x beta2. For release, replace
+  // if (dp_nominal_pos > Modelica.Constants.eps) then with
+  // if computeFlowResistance then
+  final parameter Real k = if (dp_nominal_pos > Modelica.Constants.eps) then
```
Comment (JayHuLBL) [inline Buildings/Fluid/FixedResistances/PressureDrop.mo:16]: Should keep the `if (dp_nominal_pos > Modelica.Constants.eps) then`, or change back to `if computeFlowResistance then` for merging to master?

Comment (mwetter) [reply Buildings/Fluid/FixedResistances/PressureDrop.mo]: Thanks, this should use `computeFlowResistance` which I fixed in https://github.com/lbl-srg/modelica-buildings/pull/2212/commits/2a603c2afa94e97f27f82ca4569f1c402c05ff93

**Review by JayHuLBL — APPROVED** [review]
@mwetter I think they are in good shape and can be merged.

**Pull request merged**